.github/workflows/ttnn-run-sweeps-new.yaml

```diff
@@ -0,0 +1,49 @@
+name: "ttnn - Run sweeps new"
+
+on:
+  # schedule:
+  #   - cron: "0 1,7,13,19 * * *"
+  workflow_dispatch:
+  workflow_call:
+  push:
+    branch:
+      - jdesousa/sweeps-ci
+
+jobs:
+  build-artifact:
+    uses: ./.github/workflows/build-artifact.yaml
+    secrets: inherit
+
+  ttnn-sweeps:
+    needs: build-artifact
+    strategy:
+      # Do not fail-fast because we need to ensure all tests go to completion
+      # so we try not to get hanging machines
+      fail-fast: false
+      matrix:
+        arch: [grayskull, wormhole_b0]
+    env:
+      TT_METAL_ENV: ${{ vars.TT_METAL_ENV }}
+      ARCH_NAME: ${{ matrix.arch }}
+      ELASTIC_USERNAME: ${{ secrets.SWEEPS_ELASTIC_USERNAME }}
+      ELASTIC_PASSWORD: ${{ secrets.SWEEPS_ELASTIC_PASSWORD }}
+    environment: dev
+    runs-on: ${{ matrix.arch }}
+    steps:
+      - uses: tenstorrent-metal/metal-workflows/.github/actions/checkout-with-submodule-lfs@v2.0.0
+      - name: Set up dynamic env vars for build
+        run: |
+          echo "TT_METAL_HOME=$(pwd)" >> $GITHUB_ENV
+      - uses: actions/download-artifact@v4
+        with:
+          name: TTMetal_build_${{ matrix.arch }}
+      - name: Extract files
+        run: tar -xvf ttm_${{ matrix.arch }}.tar
+      - uses: ./.github/actions/install-python-deps
+      - name: Run ttnn sweeps
+        timeout-minutes: 720
```

Comment:
please adjust timeout-minutes to a proper length after all CI tests pass with some buffer time.
Typically an extra 50% of current total time to completion is a decent rule of thumb

Reply:
Addressed in Raymond's comment below.